### ISSUE TITLE
Redirects to new docs pages do not work

### ISSUE THREAD

**Issue by nflaig** [ChainSafe/lodestar#6885]
As noted in https://github.com/ChainSafe/lodestar/commit/966f3acc0f615b62b082176412e85f7dbbb8d696#r143045200, it seems like the redirect do not work as expected.

I also found a few warnings related to that when running `docusaurus build`

```
[WARNING] @docusaurus/plugin-client-redirects: some redirects would override existing paths, and will be ignored:
- {"from":"/run/beacon-management/beacon-cli","to":"/beacon-management/beacon-cli"}
- {"from":"/run/bootnode/bootnode-cli","to":"/bootnode/bootnode-cli"}
- {"from":"/libraries/lightclient-prover/lightclient-cli","to":"/lightclient-prover/lightclient-cli"}
- {"from":"/run/validator-management/validator-cli","to":"/validator-management/validator-cli"}
```

**Comment by nflaig** [comment]
While https://github.com/ChainSafe/lodestar/pull/6940 fixed redirects locally, they are still broken when docs are deployed to https://chainsafe.github.io/lodestar/. Needs further investigation

**Comment by nflaig** [comment]
Questionable if we need those redirects anyways, looking at external references that are broken e.g. on https://launchpad.ethereum.org/en/lodestar, most of those still reference pages before initial restructuring we did way earlier, and those are not handled by the docusaurus redirects.

**Comment by nflaig** [comment]
@lodekeeper can you investigate if this is fixed now

**Comment by lodekeeper** [comment]
I checked the current state after the latest deploy:

**Current doc structure:** All pages live under `/run/...` (e.g., `/run/beacon-management/beacon-cli`, `/run/getting-started/installation`). These all return 200 ✅

**Redirect config** in `docusaurus.config.ts` creates redirects FROM `/run/<existingPath>` TO `<existingPath>`. Since the pages are already under `/run/`, the redirect generates doubled paths like `/run/run/beacon-management/...` — which means:
- The original "override existing paths" warning is gone (no conflict in latest build ✅)
- But the redirects are effectively no-ops since nobody links to `/run/run/...`

**Paths without `/run/` prefix** (e.g., `/beacon-management/beacon-cli`, `/getting-started/installation`) return 404, but that seems expected since the docs restructuring put everything under `/run/`.

As Nico noted in the last comment, the external references (e.g., launchpad.ethereum.org) likely point to even older paths from before the initial restructuring, which these redirects wouldn't cover anyway.

**TL;DR:** The redirect override warnings from the original issue are gone. The redirect config itself is mostly dead code at this point — the redirects it generates (`/run/run/...`) don't serve any purpose. Could be cleaned up but isn't actively breaking anything.
